rotkehlchen/api/v1/schemas.py

```diff
@@ -3294,9 +3286,9 @@ class RpcAddNodeSchema(Schema):
     blockchain = BlockchainField(required=True, exclude_types=(SupportedBlockchain.ETHEREUM_BEACONCHAIN,))  # noqa: E501
     name = fields.String(
         required=True,
-        validate=webargs.validate.NoneOf(
```

Comment:
i don't think this should change.

Reply:
We don't allow touching etherscan anymore. It won't be available in the UI

Reply:
i see.